src/query/ast/src/ast/statements/auto_increment.rs

```diff
@@ -0,0 +1,44 @@
+// Copyright 2021 Datafuse Labs
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::fmt::Display;
+use std::fmt::Formatter;
+
+use derive_visitor::Drive;
+use derive_visitor::DriveMut;
+
+#[derive(Debug, Clone, PartialEq, Drive, DriveMut)]
+pub struct AutoIncrement {
+    pub start: u64,
+    pub step: u64,
+    pub is_order: bool,
+}
+
+impl AutoIncrement {
+    pub fn sequence_name(database: &str, table_id: u64, column_id: u32) -> String {
+        format!("_sequence_{database}_{table_id}_{column_id}")
```

Comment:
If sequence_name store database, please use database_id.

But table_id is unique, so there's no need to store database.